types_executor.go

```diff
@@ -229,12 +232,13 @@ func (v *Venom) RunUserExecutor(ctx context.Context, runner ExecutorRunner, tcIn
 		TestCaseInput: TestCaseInput{
 			Name:         ux.Executor,
 			RawTestSteps: ux.RawTestSteps,
-			Vars:         vrs,
+			Vars:         inputOnlyVrs,
```

Comment:
`tc.Vars == tc.TestCaseInput.Vars.`

I don't understand the goal of the variable inputOnlyVrs: Doing `TestCaseInput.Vars = intputOnlyVrs`, and then `tc.Vars.AddAll(vrs)` is the same as the previous code.